src/Security/Security.php

```diff
@@ -117,17 +117,13 @@ public static function getPermissionsSQLByUserId(int $owner_id, bool $accessible
 		if ($local_user && $local_user == $owner_id) {
 			$sql = '';
 		/*
-		 * Authenticated visitor. Load the groups the visitor belongs to.
+		 * Authenticated visitor. Load the circles the visitor belongs to.
 		 */
 		} elseif ($remote_contact) {
 			$gs = '<<>>'; // should be impossible to match
 
-			$groups = Group::getIdsByContactId($remote_contact);
-
-			if (is_array($groups)) {
-				foreach ($groups as $g) {
-					$gs .= '|<' . intval($g) . '>';
-				}
+			foreach (Circle::getIdsByContactId($remote_contact) as $g) {
```

Comment:
`$g` comes from group, if we follow naming-convention, then this should be renamed at least to `$c` or even `$circle`.

Reply:
Good catch!